packages/fxa-settings/src/pages/SigninBounced/en.ftl

```diff
@@ -0,0 +1,6 @@
+signin-bounced-header = Sorry. We’ve locked your&nbsp;account.
+# $email (string) - The user's email.
+signin-bounced-message = The confirmation email we sent to { $email } was returned and we’ve locked your account to protect your Firefox data.
+signin-bounced-help = <p>If this is a valid email address, <a data-l10n-name="supportLink">let us know</a> and we can help unlock your account.</p>
```

Comment:
Are the `<p>` elements necessary in the string here? Looking through current ftl strings I don't see any examples where these are used.

Reply:
oh interesting! I needed to put the `<p>` tags into the markup because of the variable, but it looks like it can be removed from the ftl file -- removing now!